File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.java
class Solution {
    public int removeDuplicates(int[] nums) {
     
        int temp = 0;

        for(int i=1;i<nums.length;i++){
            if(nums[i-1]!=nums[i]){
                temp++;
            nums[temp] = nums[i];
        }
    }
    return temp+1;
}
}

File: Data Structures & Algorithms/reorder-linked-list/submission-0.py
# Definition for singly-linked list.
# class ListNode:
#     def __init__(self, val=0, next=None):
#         self.val = val
#         self.next = next

class Solution:
    def reorderList(self, head: Optional[ListNode]) -> None:
        curr = head
        fast = slow = head

        while fast and fast.next:
            fast = fast.next.next
            slow = slow.next
        #Now slow is in middle
        temp = slow.next
        slow.next = None
        
        prev = None

        while temp:
            nxt = temp.next
            temp.next = prev
            prev = temp
            temp = nxt
        
        while prev and curr:
            nxt1 = curr.next
            nxt2 = prev.next

            curr.next = prev
            prev.next = nxt1
            prev = nxt2
            curr = nxt1